src/hivpy/hiv_status.py

```diff
@@ -14,6 +15,14 @@
 
 from . import output
 from .common import COND, SexType, opposite_sex, rng, timedelta
+from .prep import PrEPType
+
+
+# Ab (default), PCR (RNA VL), Ag/Ab
+class HIVTestType(IntEnum):
+    Ab = 0
+    PCR = 1
+    AgAb = 2
```

Comment:
Can we add comments for what these are 
@andrew-phillips-1 any useful information to provide for these acronyms?

Reply:
I suggest renaming PCR as NA for Nucleic Acid-based test

Ab - antibody only assay
NA - nucleic acid based 
AgAb - Antigen/antibody combined assay

We could change the order to 

Ab = 0
AgAb = 1
NA = 2

so that the higher the number the higher the sensitivity

Reply:
Do we care about ordering by sensitivity more than ordering by how commonly you expect the test type to be used during simulations?

Reply:
ok yes that's fine to keep as is then